Backend/feed/feed.go

```diff
@@ -0,0 +1,380 @@
+package main
+
+import (
+	"database/sql"
+	"strconv"
+
+	_ "github.com/go-sql-driver/mysql"
+)
+
+func main() {
+	println("get by post num: \n")
+	test := getPostData(1)
+	println(test)
+
+	initialPostCount := 3
+
+	println("\n\n\nStarting random sort: \n")
+	var testarr []string
+	var testdupe []int
+	testarr, testdupe = initialFeedByRandom(initialPostCount)
+	displayPost(testarr)
+	testarr = feedByRandom(testdupe)
+	displayPost(testarr)
+
+	println("\n\n\nStarting time sort: \n")
+
+	testarr = initialFeedByTime(initialPostCount)
+	displayPost(testarr)
+	testarr = feedByTime(initialPostCount)
+	displayPost(testarr)
+}
+
+// func to get individual post
+func getPostData(post_id int) string {
+	dsn := "capstone:csc450@tcp(71.89.73.28:3306)/capstone"
+
+	// Open a connection to the database
+	db, err := sql.Open("mysql", dsn)
+	if err != nil {
+		panic(err)
+	}
+	defer db.Close()
+
+	// Ping the database to verify the connection is alive
+	err = db.Ping()
+	if err != nil {
+		panic(err)
+	}
```

Comment:
I think when it comes to connecting to the database it might be nice to have like a reusable utils function. I think we will end up with a lot of functions that connect to the database